qa/rpc-tests/blockdelay.py

```diff
@@ -23,35 +23,9 @@ def setup_chain(self, split=False):
         print("Initializing test directory "+self.options.tmpdir)
         initialize_chain_clean(self.options.tmpdir, 5)
 
-    def setup_network(self, split=False):
-        self.nodes = []
-
-        # -exportdir option means we must provide a valid path to the destination folder for wallet backups
-        ed0 = "-exportdir=" + self.options.tmpdir + "/node0"
-        ed1 = "-exportdir=" + self.options.tmpdir + "/node1"
-        ed2 = "-exportdir=" + self.options.tmpdir + "/node2"
-        ed3 = "-exportdir=" + self.options.tmpdir + "/node3"
+    def setup_nodes(self):
```

Comment:
It seems to me there is no need to override `setup_nodes` with this specific function (having the exact same implementation).

Reply:
The default is 4 nodes, Here are 3.

Reply:
Mhm, I see 4 nodes in the overriding:
```
def setup_nodes(self):
        self.nodes = start_nodes(4, self.options.tmpdir)
```
so it seems to me it's the same implementation.